first.c

```diff
@@ -0,0 +1,112 @@
+
+#include <stdio.h>
+#include <ctype.h>
+#include <stdlib.h>

```

Comment:
is this header file required?

Reply:
It was a mistake to include <stdlib.h> here. This library is mainly used for memory allocation, but I didn't use those functions in this program. I'll remove it